xml/sle_update_upgrading.xml

```diff
@@ -67,26 +67,45 @@ Check with Simona, if this is supported or needed.
   </sect2>
 
   <sect2 xml:id="sec.update.prep.relnotes">
-   <title>Check the Release Notes</title>
+   <title>Read the Release Notes</title>
    <para>
-    In the release notes you can find additional information on what has
-    changed since the previous release of &sle;. If you are skipping one
-    or more Service Packs, also check the release notes of the skipped
-    Service Packs for changes made between each release. The release
-    notes usually only contain the changes from the last to the current
-    release.
-    Verify there if your specific hardware or setup needs special
-    considerations, which of your favorite specific software packages
-    have changed significantly, and which precautions you should take in
-    addition to the general recommendations of this section. The release
-    notes also provide last-minute information and known issues that
-    could not make it to the manual on time.
+    In the release notes you can find additional information on changes
+    since the previous release of &productname;. Check the release notes
+    to see whether:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      your hardware needs special considerations.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      any used software packages have changed significantly.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      special precautions are necessary for your installation.
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    The release notes also provide information that could not make it
+    into the manual on time. They also contain notes about known issues.
+   </para>
+   <para>
+    If you are skipping one or more Service Packs, check the release
+    notes of the skipped Service Packs as well. The release notes
+    usually only contain the changes from the last to the current
+    release. You can miss important changes if you are only reading the
```

Comment:
More precisely: `... usually only contain the changes between two subsequent releases.`?